docs/excel/excel-add-ins-workbooks.md

```diff
@@ -181,6 +181,41 @@ Excel.run(function (context) {
 }).catch(errorHandlerFunction);
 ```
 
+## Access application culture settings (preview)
+
+A workbook has language and culture settings that affect how certain data is displayed. This settings can help localize data when your add-in user-base is using shared workbooks across different regions. Your add-in can use string parsing to standardize the format of numbers, dates, and times based on the system culture settings.
+
+`Application.cultureInfo` defines the system culture settings as a [CultureInfo](/javascript/api/excel/excel.cultureinfo) object. This contains settings like the numerical decimal separator or the date format.
+
+Some culture settings can be [changed through the Excel UI](https://support.office.com/article/Change-the-character-used-to-separate-thousands-or-decimals-c093b545-71cb-4903-b205-aebb9837bd1e). The system settings are preserved in the `CultureInfo` object. Any local changes are kept as [Application](/javascript/api/excel/excel.application)-level properties, such as `Application.decimalSeparator`.
+
+The following sample changes the decimal separator character of a numerical string from a ',' to the character used by the system settings.
+
+```js
+// This will convert a number like "14,37" to "14.37"
+// (assuming the system decimal separator is ".").
+Excel.run(function (context) {
+    var sheet = context.workbook.worksheets.getItem("Sample");
+    var decimalSource = sheet.getRange("B2");
+    decimalSource.load("values");
+    context.application.cultureInfo.numberFormat.load("numberDecimalSeparator");
+
+    return context.sync().then(function() {
+        var systemDecimalSeparator =
+            context.application.cultureInfo.numberFormat.numberDecimalSeparator;
+        var oldDecimalString: string = decimalSource.values[0][0];
```

Comment:
Is this line mixing TypeScript (type declared as string) and JavaScript ("var")?

Reply:
Good catch. Fixed!